website/docs/sdk-reference/android.md

```diff
@@ -476,6 +476,17 @@ ConfigCatClient client = ConfigCatClient.get("#YOUR-SDK-KEY#", options -> {
     options.cache(new MyCustomCache()); // inject your custom cache
 });
 ```
+### Platform specific cache
+
+The SDK provides the `ConfigCatPreferencesCache` as platform specific cache. The cache uses the Android `SharedPreferences` to store the downloaded `config.json`.
+
+It has a dependency on `android.content.Context`, so it won't be enabled by default, but it can be explicitly set by providing an appropriate `Context`.
```

Comment:
This sentence has 3 'it's. I don't know which refers to what. I suggest rephrasing.

Reply:
Rephrased.